indico/core/celery/flower.py

```diff
@@ -69,9 +69,10 @@ def get_auth_http_client(self):
 
     @asynchronous
     def get(self):
-        redirect_uri = 'http{}://{}:{}/login'.format('s' if 'ssl_options' in settings else '',
-                                                     options.address or 'localhost',
-                                                     options.port)
+        redirect_uri = 'http{}://{}{}{}/login'.format('s' if 'ssl_options' in settings else '',
+                                                      options.address or 'localhost',
+                                                      ':' + options.port if not options.unix_socket else '',
```

Comment:
The port should only be added if it's not the default (80/443 depending on the protocol)

Reply:
The patch removes adding the port (which is 5555 by default for flower) if flower is actually listening on a unix socket. What's not handled currently is an upstream server on non-default ports in this case.  As the options are handled by flower this is currently out of scope.

Reply:
Oh right we were already always adding the port before. Nevermind then.